functional_tests/common/scvmm_utils.py

```diff
@@ -236,8 +192,8 @@ def create_instance(morpheus_session, instance_name=None, group_id=None, cloud_i
 
         # Generate payload
         log.info("Generating instance payload...")
-        create_instance_payload = SCVMMUtils.get_create_instance_payload(
-            instance_name=instance_name, template= template_id, group_id=group_id, cloud_id=cloud_id, host_id=host_id
+        create_instance_payload = SCVMMpayloads.get_create_instance_payload(
+            morpheus_session, instance_name=instance_name, template= template_id, group_id=group_id, cloud_id=cloud_id, host_id=host_id
```

Comment:
The template_id should be accepted as an argument as the create_instance is a generic method.